tools/spec-gen-sdk/ci.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - tools/spec-gen-sdk
+
+pr:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - tools/spec-gen-sdk
+
+parameters:
+  - name: Publish
```

Comment:
Is there a reason you want to have this publish parameter? It will not actually control the real publishing at https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/stages/archetype-sdk-publish-js.yml#L24. 

I would suggest removing it.